omod/src/main/java/org/openmrs/module/oauth2login/web/CustomDispatcherServlet.java

```diff
@@ -40,8 +40,7 @@ public void setServletContext(ServletContext servletContext) {
 		
 		try {
 			XmlWebApplicationContext appContext = new XmlWebApplicationContext();
-			appContext.setConfigLocation("classpath:webModuleApplicationContext.xml");
-			
+			appContext.setConfigLocation("classpath*:/oauth2login/webApplicationContext.xml");
```

Comment:
Don't you want to use [this](https://github.com/openmrs/openmrs-module-oauth2login/blob/37183a76108384eb0484fa509a2ed890619014a3/api/src/main/java/org/openmrs/module/oauth2login/OAuth2LoginConstants.java#L19)?

Reply:
Resolved, thanks @mks-d.